torchchat/cli/convert_hf_checkpoint.py

```diff
@@ -130,31 +136,34 @@ def load_safetensors():
                 continue
         assert state_dict is not None, f"Unable to load tensors from {file}"
         merged_result.update(state_dict)
+
     final_result = {}
     for key, value in merged_result.items():
         if "layers" in key:
             abstract_key = re.sub(r"(\d+)", "{}", key)
             layer_num = re.search(r"\d+", key).group(0)
-            new_key = weight_map[abstract_key]
-            if new_key is None:
-                continue
+            new_key = weight_map.get(abstract_key, abstract_key)
             new_key = new_key.format(layer_num)
         else:
-            new_key = weight_map[key]
+            new_key = weight_map.get(key, key)
```

Comment:
I might be misreading something, but why are we keeping keys with missing entries

Reply:
This is probably not fully needed now that I understand how the mapping is working better. I added this when trying to get it working since Granite Code had missing entries. I was thinking at the time that unconverted tensor names would be treated in a generic way post-conversion, but I now realize that's not the case. I'll remove this change.